### ISSUE TITLE
Fast Mode

### ISSUE THREAD

**Issue by Trucidar** [JonasNilson/idle_master_extended#23]
I noticed my fast mode is idling 30 games at a time, but when it goes to run individually, instead of running those 30, it looks like it runs my entire library of games with card drops (1800). Is there anyway to avoid this?

**Comment by JonasNilson** [comment]
@Trucidar I'll have to take a look at the code, but I have a feeling that the individual idling is based on the full list of games 🤔

**Comment by JonasNilson** [comment]
@Trucidar I'm sorry for not updating. I found a place in the code that has potential for an update to follow the suggestion you have, but atm I have no way to test it properly. 

I'll keep this issue open until I do.

**Comment by Trucidar** [comment]
All good. I appreciate you looking into it. I imagine it's only useful to people who are running the tool for the first while and have more than 30 available games in their inventory.

**Comment by JonasNilson** [comment]
I've made a change that should help with your issue:
* https://github.com/JonasNilson/idle_master_extended/commit/86533705fc82404e814664f253605658bcb3a77e

I would appreciate if you could test it out through Visual Studio

**Comment by JonasNilson** [comment]
> All good. I appreciate you looking into it. I imagine it's only useful to people who are running the tool for the first while and have more than 30 available games in their inventory.

It should work now: https://github.com/JonasNilson/idle_master_extended/releases/tag/v1.4

**Comment by Trucidar** [comment]
Thanks for adding this! I tried it out. It's a definite improvement. It's behaviour seems a bit different, it used to idle 30 simultaneously for 15mins, then the entire collection for 5 seconds each... Now it seems to be idling around 30 games for 6mins, then 30 games for 5mins each.

I'm happy regardless.

**Comment by JonasNilson** [comment]
@Trucidar
No worries. I couldn't find a good way of testing it properly, but if you run into issues you can answer here or open a new issue

**Comment by Trucidar** [comment]
Thanks again. I tried 1.4. It's behaving oddly. Not sure how to explain it. It used to idle 30 simultaneously for 5 mins, then the whole library for 5s at a time, then 30 simultaneously for 15 mins, then whole collection for 5 seconds each... repeating.

Now it doesn't act predictably at all. Sometimes It idles 31 games for 5minutes.. sometimes 15 minutes. Sometimes it just goes straight to idling one game over and over again in 5min intervals until all the cards have dropped. Sometimes it goes thru a list of 30 games and idles each for either 5 mins or 15 minutes.

Once I did see it act normally where it ran 31 simultaneously than 1 at a time for 5seconds each. But usually it's got a mind of it's own. If it's too much a hassle that's not a problem, I'll go back to v1.2!
